src/main/scala/de/htwg/se/sudoku/Sudoku.scala

```diff
@@ -20,8 +26,12 @@ object Sudoku {
     var input: String = ""
 
     do {
-      input = readLine()
-      tui.processInputLine(input)
+      if (Console.in.ready()) {
```

Comment:
Don't know why but `Console.in` isn't always ready using docker. While starting the image, I got often a NullPointerException in the `readLine`-method...

Reply:
the else case should not be needed as `input` is initialized as an emtpy string anyway